rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/480_date_histogram.yml

```diff
@@ -0,0 +1,140 @@
+setup:
+  - do:
+      indices.create:
+        index: test
+        body:
+          settings:
+            number_of_replicas: 0
+          mappings:
+            properties:
+              date:
+                type: date_nanos
+
+  - do:
+      bulk:
+        index: test
+        refresh: true
+        body:
+          - { "index": {} }
+          - { "date": "2015-01-01" }
+          - { "index": {} }
+          - { "date": "2015-01-01T12:10:30.123456789Z" }
+          - { "index": {} }
+          - { "date": 1420070400 }
+          - { "index": {} }
+
+---
+"date_histogram on date_nanos without timezone fixed interval":
+  - skip:
+      version: " - 8.1.99"
+      reason: bug fixed in 8.2.0
```

Comment:
The linked issue lists https://github.com/elastic/elasticsearch/pull/52016 as resolving this, which landed in 7.6.1.  Let's confirm it works there or at least 7.7.0 and update the skip version appropriately.  As noted in your other PR, I'm in favor of backporting these tests to 7.17.